dev/provider_packages/refactor_provider_packages.py

```diff
@@ -730,6 +730,9 @@ def kubernetes_package_filter(node: LN, capture: Capture, filename: Filename) ->
             .rename("add_sidecar")
         )
 
+    def refactor_www(self):
+        (self.qry.select_module("airflow.www").rename("airflow.www_rbac"))
```

Comment:
Can you tell a little more about it?

Reply:
# We need to refactor www to www_rbac so that the Hooks that expose
        # custom widget fields to connection form can be imported.

Reply:
Hooks in backport packages can't expose custom fields though -- what code in the provider is this updating? I don't see anything in the providers that are doing `import airflow.www` is there?